src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -13,6 +14,7 @@ import DowntimeBanner from '../../../../platform/monitoring/DowntimeNotification
 import siteName from '../../../brand-consolidation/site-name';
 
 const loginHandler = loginType => () => {
+  Raven.setTagsContext({ loginType });
```

Comment:
Will errors still have this tag even if the user doesn't complete login? Does it get removed if they experience errors in the same session after they log out?

Reply:
Good call.  There isn't really a good way to accurately detect abandonment. I'm going to rethink this.  May just make more sense to send the `loginType` tag on error instead of setting it as a global tag.

Reply:
@U-DON I've updated the following:
* Global Raven tag context is not set until successful login
* Raven tag context is cleared upon `logout()` or `teardownProfileSession()`
* Pending `loginType` is stored in `sessionStorage` so that we can include in tags in case of login error. It is cleared after a successful/failed login.

Let me know if you think that addresses your comments